src/core/api/fims/public/_swagger.json.tpl

```diff
@@ -0,0 +1,420 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenID Provider",
+    "description": "OpenID Provider service",
+    "version": "0.0.1"
+  },
+  "host": "api.openid-provider.io.italia.it",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/oauth/authorize": {
```

Comment:
also here I suggest to add fims prefix as base path

Reply:
Agreed with @michaeldisaro